### ISSUE TITLE
Some training details.

### ISSUE THREAD

**Issue by yingkaining** [sukjunhwang/VITA#6]
I have some problems about excellent VITA. 
1. How about train without COCO joint? How much performance improvement brings by COCO joint?
2. The performance drops about 10 points when freeze detector. However, in GenVIS, the performance seems well with m2f frozen.

**Comment by miranheo** [comment]
Hi @yingkaining 

Thank you for your interest and comments!

1. It depends on the dataset and backbone, but for example YouTube-VIS 2019 with ResNet 50, we experimentally observed a drop of around 1.7 AP. (49.8 AP --> 48.1 AP)
2. In GenVIS, we freeze Mask2Former which is trained on the target VIS domain.

**Comment by yingkaining** [comment]
Thanks for your kindly reply. I want to train VITA by myself, however the R50, YTVIS2019, bs8 setting needs 16 hours on 8 A100s. How about set a lager batch size,  e.g. 16 or 32. Have you try this before ?

**Comment by miranheo** [comment]
Could you tell me what kinds of A100 you have, i.e., 40GB vs. 80GB?

**Comment by yingkaining** [comment]
40GB

**Comment by yingkaining** [comment]
BTW, It's interesting to set DATASETS.DATASET_RATIO as `coco_ratio` > `vis_datasets_ratio`. In ovis, you even set `ovis ratio`  as 0.25. Is this a reason why VITA get low performance on OVIS? Because VITA didn't fit enough occlusion cases in training.

**Comment by yingkaining** [comment]
BTW, It's interesting to set DATASETS.DATASET_RATIO as `coco_ratio` > `vis_datasets_ratio`. In ovis, you even set `ovis ratio`  as 0.25. Is this a reason why VITA get low performance on OVIS? Because VITA didn't fit enough occlusion cases in training.

**Comment by miranheo** [comment]
1. We haven't experimented with it, but since the optimizer is AdamW, you may keep the learning rate and reduce the original scheduling in half while doubling batch size (8 —> 16). It may not be optimal because it is AdamW.
2. This is because existing VIS datasets are vulnerable to the overfitting problem due to the small number of videos. Although OVIS has a longer sequence, it has only 607 training videos. It is much smaller than YouTube-VIS, which has more than 2,000 videos.

**Comment by miranheo** [comment]
We have various understandings of the VITA's low performance on OVIS, but we can say that there is a lack of explicit temporal modeling for one reason. We assume that the effect of frame-level spatial occlusion is small because the detector shows already robust performance on independent frames. This is demonstrated in the performance of recent per-frame online models (MinVIS, IDOL).

**Comment by yingkaining** [comment]
Thanks for your valuable shareing.

**Comment by miranheo** [comment]
We sincerely appreciate your questions. If you have any further questions feel free to contact us :)
miran@yonsei.ac.kr, sj.hwang@yonsei.ac.kr
